cmd/kubelet/app/plugins.go

```diff
@@ -41,6 +42,6 @@ func ProbeVolumePlugins() []volume.Plugin {
 	allPlugins = append(allPlugins, git_repo.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, secret.ProbeVolumePlugins()...)
-
+	allPlugins = append(allPlugins, iscsi_pd.ProbeVolumePlugins()...)
```

Comment:
Is "_pd" a useful suffix?  We use it on gce_pd because that is literally the product name "Persistent Disk".  Could we just call this pkg "iscsi" ?